main/ertp/api/amount-math.md

```diff
@@ -126,128 +91,153 @@ If not valid, throws an exception. This checks if
-on whether the `amountMath` is `MathKind.NAT` (`0`), `MathKind.SET` (`[]`), 
-or `MathKind.STRING_SET` (`[]`).
+on whether the `amountMathKind` is `MathKind.NAT` (`0`) of `MathKind.SET` (`[]`).
 
 ```js
-// Returns an empty amount for this amountMath.
-// Since this is a fungible amount it returns an amount
+// Returns an empty amount.
+// Since this is a fungible amountKind it returns an amount
 // with 0n as its value.
-const empty = quatloosAmountMath.getEmpty();
+const empty = amountMath.makeEmpty(quatloosBrand, mathKind.NAT);
+```
+
+## amountMath.makeEmptyFromAmount(amount)
+- `amount` `{Amount}`
+- Returns: `{Amount}`
+
+Return the `amount` representing an empty amount, using another
+`amount` as the template for the `brand` and `mathKind`.
+
+```js
+// quatloosAmount837 = { value: 837n, brand: quatloos }
+const quatloosAmount837 = amountMath.make(quatloosBrand, 837n);
+// Returns an amount = { value: 0n, brand: quatloos }
+const quatloosAmount0 = amountMath.makeEmptyFromAmount(quatloosAmount837);
 ```
 
-## amountMath.isEmpty(amount)
+## amountMath.isEmpty(amount, brand?)
 - `amount` `{Amount}`
+- `brand?` `{Brand}` (optional, defaults to `undefined`)
 - Returns: `{Boolean}`
 
 Returns `true` if the `amount` is empty. Otherwise returns `false`.
 
+The `brand` parameter is optional, and defaults to `undefined`.
+
 ```js
-const empty = quatloosAmountMath.getEmpty();
-const quatloos1 = quatloosAmountMath.make(1n);
+const empty = amountMath.makeEmpty(quatloosBrand, MathKind.NAT);
+const quatloos1 = amountMath.make(quatloosBrand, 1n);
 
 // returns true
-quatloosAmountMath.isEmpty(empty)
+const result = amountMath.isEmpty(empty);
 
 // returns false
-quatloosAmountMath.isEmpty(quatloos1)
+const result = amountMath.isEmpty(quatloos1);
 ```
 
-## amountMath.isGTE(leftAmount, rightAmount)
+## amountMath.isGTE(leftAmount, rightAmount, brand?)
 - `leftAmount` `{Amount}`
 - `rightAmount` `{Amount}`
+- `brand` `{Brand}` (optional, defaults to `undefined`)
 - Returns: `{boolean}`
 
 Returns `true` if the `value` of `leftAmount` is greater than or equal to
 the `value` of `rightAmount`. Both `amount` arguments must have the same
-`brand` as this `amountMath`.
+`brand`.
+
+The `brand` argument is optional, defaulting to `undefined`.
 
 For non-fungible `values`, what "greater than or equal to" is depends on the 
 kind of `amountMath`. For example, { 'seat 1', 'seat 2' } is considered
 greater than { 'seat 2' } because the former both contains all of the latter's 
 contents and has additional elements.
 
 ```js
-const empty = quatloosAmountMath.getEmpty();
-const quatloos5 = quatloosAmountMath.make(5n);
-const quatloos10 = quatloosAmountMath.make(10n);
+const empty = amountMath.makeEmpty(quatloosBrand, MathKind.NAT);
+const quatloos5 = amountMath.make(quatloosBrand, 5n);
+const quatloos10 = amountMath.make(quatloosBrand, 10n);
 
 // Returns true
-quatloosAmountMath.isGTE(quatloos5, empty);
+amountMath.isGTE(quatloos5, empty);
```

Comment:
It'd be good to include an example that makes use of the optional absolute brand enforcement:

`amountMath.isGTE(empty, quatloos5, quatloosBrand);`